debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -594,7 +594,7 @@ protected Object convertTimestampWithZone(Column column, Field fieldDefn, Object
         else if (data instanceof java.util.Date) {
             // any Date like subclasses will be given to us by the JDBC driver, which uses the local VM TZ, so we need to go
             // back to GMT
-            data = OffsetDateTime.ofInstant(Instant.ofEpochMilli(((Date) data).getTime()), ZoneOffset.UTC);
```

Comment:
@Naros, can you check out the `convertTimeWithZone()` right below this one, too? I suppose it's prone to the same issue.

Reply:
@gunnarmorling I tried that but it caused an `UnsupportedOperationException` since postgres gives us a `java.sql.Time` instance for the value and you cannot call `toInstant()`.

Reply:
I don't mind too much either way, as long as it's addressed one way or another :)